apps/web/docker/Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM latitudedata/base AS base
+FROM node:20-alpine AS base
```

Comment:
With only these changes the image builds. I find it better to have an install of dependencies for build and then another stage for `--prod` dependencies. Take into account if you change this